coverage-tests.js

```diff
@@ -1306,30 +1289,34 @@ test('should send dom and location when check region by selector fully with cust
-    driver.executeScript('window.scrollTo(0, 350)') // not really needed
     driver.click('#centered')
-    driver.executeScript('document.documentElement.style.overflow="hidden";') // TODO this is due to differences between JS and non-JS SDK's. Since what's important in this test is not the image itself, nor can this be solved with branch baselines, we hard coded hide scrollbars on the HTML element in order to verify that the DOM and location are correct.
-    driver.executeScript('document.querySelector("#modal-content").setAttribute("data-applitools-target", "true");')
-    const scrollRootElement = driver.findElement('#modal1').ref('scrollRootElement')
-    driver.executeScript('arguments[0].setAttribute("data-applitools-expected-scroll", "true");', scrollRootElement)
-    eyes.check({region: '#modal-content', isFully: true, scrollRootElement})
-    const result = eyes.close(false).ref('result')
-    const info = helpers.getTestInfo(result).ref('info')
+    // TODO this is due to differences between JS and non-JS SDK's. Since what's important in this test is not the image itself, nor can this be solved with branch baselines, we hard coded hide scrollbars on the HTML element in order to verify that the DOM and location are correct.
+    // TODO this happens because non-JS emitters are ignore env args
+    driver.executeScript('document.documentElement.style.overflow="hidden";')
+    const element = driver.findElement('#modal-content')
+    driver.executeScript('arguments[0].setAttribute("data-expected-target", "true");', element)
+    const scrollRootElement = driver.findElement('#modal1')
+    driver.executeScript('arguments[0].setAttribute("data-expected-scroll", "true");', scrollRootElement)
+    eyes.check({region: element, isFully: true, scrollRootElement})
+    const result = eyes.close(false)
+    const info = helpers.getTestInfo(result)
     assert.equal(info.actualAppOutput[0].image.location, {x: 112, y: 0})
     assert.equal(info.actualAppOutput[0].image.hasDom, true)
-    const dom = helpers.getDom(result, info.actualAppOutput[0].image.domId).ref('dom')
-    const targetElement = dom.getNodesByAttribute('data-applitools-target').ref('targetElement')
-    assert.equal(targetElement.length, 1)
-    assert.equal(targetElement[0].rect.top, 0)
-    assert.equal(targetElement[0].rect.left, 112)
-    const scrollingElements = dom.getNodesByAttribute('data-applitools-scroll').ref('scrollingElements')
+    const dom = helpers.getDom(result, info.actualAppOutput[0].image.domId)
+    const scrollingElements = dom.getNodesByAttribute('data-applitools-scroll')
     assert.equal(scrollingElements.length, 1)
     assert.equal(scrollingElements[0].attributes['data-applitools-scroll'], 'true')
-    assert.equal(scrollingElements[0].attributes['data-applitools-expected-scroll'], 'true')
+    assert.equal(scrollingElements[0].attributes['data-expected-scroll'], 'true')
+    assert.equal(helpers.math.round(scrollingElements[0].rect.left), 0)
```

Comment:
why is this important to verify?

Reply:
Because dom should contain an initial position of the scrolling element.